graphing.py

```diff
@@ -28,7 +28,7 @@ def _get_chunk_size(self):
     
     def _reduce_to_avg(self, list: list, chunk_size: int):
         """Divides a long list of values into chunks and finds the average value of each chunk."""
-        chunks = [list[i:i + chunk_size] for i in range(0, len(list), chunk_size)]
+        chunks = [list[i:(i + chunk_size)] for i in range(0, len(list), chunk_size)]
         return [sum(chunks[i]) / len(chunks[i]) for i in range(len(chunks))]
```

Comment:
_reduce_to_best has for i in range(len(chunks) - 1) - this should be the same for both methods - pick one!